src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx

```diff
@@ -170,10 +170,7 @@ const LocationFiltersForm = (props: Props & InjectedWizardProps) => {
                     shouldCollapse
```

Comment:
One more thing. This is poor design to couple `SummaryListItem` and `CollapsibleList` together. I would remove any of the collapsible nonsense out of `SummaryListItem` then just create `CollapsibleList`s as children i.e.

```typescript
<SummaryList>
  <SummaryListItem term="Whatever">
    <CollapsibleList>
      ...
    </CollapsibleList>
  <SummaryListItem>
</SummaryList>
```

This should probably always reveal other locations where we're using `div`s in `SummaryListItems`.

Tip: If you ever find yourself adding random props that have nothing to do with the component's concern just to add on little features like this, you can pretty much always achieve the same thing using just `children` or render props.